common/elasticsearch/src/test/scala/uk/ac/wellcome/elasticsearch/WorksIndexTest.scala

```diff
@@ -116,7 +115,7 @@ class WorksIndexTest
     implicit encoder: Encoder[T]): Future[Response[IndexResponse]] =
     elasticClient
       .execute {
-        indexInto(index.name / index.name).doc(toJson(t).get)
+        indexInto(index.name).doc(toJson(t).get)
```

Comment:
Is this changing the name of the index, and if so, does that matter?

Reply:
Na, that's the syntax for `{index_name}/{doc_type}`.
We were previously giving the `doc_type` the same value as `index_name`.